### PR TITLE
Test case for calculate row column limits

### PR DESCRIPTION
**Pull request by fnwaohaHC**
-Added a testcase for calculate row column limits
-Also refactored the method into table tools so a table object is not needed to call it. But, the original method is still preserved in table class because other things rely on it.

### CODE REVIEW

File: table_processing/table_tools.py [hunk 1]
```diff
@@ -40,3 +40,44 @@ def remove_duplicate_limits(limit_list, threshold):
...
+
+    return(column_limits,row_limits)
+    
+
+def _calculate_row_column_limits(image_dim, bbox_list):
+    column_limits, row_limits =_calculate_raw_row_column_limits(bbox_list)
+    threshold = 2  # no row or column can be only 2 pixels wide
+    row_limits = remove_duplicate_limits(row_limits, threshold)
+    column_limits = remove_duplicate_limits(column_limits, threshold)
+
```
Comment (Louis-Boulet-HC) [inline table_processing/table_tools.py:66]: we should look to tune this once merged to reflect the changes made to the  _calculate_row_column_limits() function in Table.py from the previous recent PRs

**Review by Louis-Boulet-HC — APPROVED** [review]

**Review by zgaffarHC — APPROVED** [review]

File: table_processing/table_tools.py (hunk 1 above)
Comment (fnwaohaHC) [reply table_processing/table_tools.py:66]: Ya this is gonna have to be tweaked to reflect the changes you guys made in the last pr once merged

Comment (pgaviganHC) [reply table_processing/table_tools.py:66]: I suggest a rebase from main and rerun to confirm that things are looking good.

**Review by pgaviganHC — COMMENTED** [review]
Assuming that the test is still good after a rebase then I think this is good to merge

**Comment by fnwaohaHC** [comment]
branch is up to date and test passes. Merging

**Pull request merged**